bin/add-server

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+require('dotenv').config();
+const axios = require('axios');
+
+// step 1, check command line args
+const args = process.argv.slice(2); // first 2 are "node add-server"
+
+const newServer = {};
+
+for (let i = 0 ; i < args.length ; i++) {
+  const argKey = args[i];
+  const argValue = args[++i];
+
+  switch (argKey) {
+    case '-e':
+    case '--endpoint':
+      newServer['endpoint'] = argValue;
+      break;
+
+    case '-t':
+    case '--type':
+      newServer['type'] = argValue;
+      break;
+
+    case '-c':
+    case '--clientId':
+      newServer['clientId'] = argValue;
+      break;
+
+    case '-n':
+    case '--name':
+      newServer['name'] = argValue;
+      break;
+
+    default:
+      console.error(`Unknown command line option: ${argKey}`);
+      console.error(`Full args: ${args}`);
+      process.exit(1);
+  }
+}
+
+if (!newServer['endpoint']) {
+  console.error("Required argument -e|--endpoint missing!");
+  process.exit(1);
+}
```

Comment:
The required fields also include `type` and `clientId`. The app won't work correctly without them